e2e/package.json

```diff
@@ -23,7 +23,6 @@
     "@graphql-debugger/trace-schema": "workspace:^",
     "@graphql-debugger/data-access": "workspace:^",
     "puppeteer": "21.1.1",
-    "@types/puppeteer": "7.0.4",
```

Comment:
I noticed we removed this.

I get no build or IDE errors so they must now be included 😎

Reply:
https://www.npmjs.com/package/@types/puppeteer Deprecated for a while now, so I removed them :)